### PR TITLE
Added _supportedLayout

### LINKED ISSUE #74: _supportedLayout attribute needs adding to schema
This is to help the authoring tool determine layout styles for this component

### PR DESCRIPTION
**Pull request by darylhedley**
fixes #74

### CODE REVIEW

**Comment by dancgray** [comment]
+1

**Comment by hrajotia** [comment]
+1

File: js/adapt-contrib-media.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ define(function(require) {
         preRender: function() {
             this.listenTo(Adapt, 'device:resize', this.onScreenSizeChanged);
             this.listenTo(Adapt, 'device:changed', this.onDeviceChanged);
+            this.listenTo(this.model, 'change:_isCompelte', this.removeInviewListener);
```
Comment (dennis-learningpool) [inline js/adapt-contrib-media.js]: Spelling error: should be `change:_isComplete`?

**Comment by dennis-learningpool** [comment]
+1

**Comment by hrajotia** [comment]
+1

**Comment by oliverfoster** [comment]
+1

**Comment by taylortom** [comment]
+1

**Comment by hrajotia** [comment]
+1

**Pull request merged**